web/srv/api_handlers.go

```diff
@@ -243,7 +240,8 @@ func (h *handler) handleAPITap(w http.ResponseWriter, req *http.Request, p httpr
 			// socket with `ClosePolicyViolation` status code so that the error
 			// renders without the error prefix in the banner
 			if httpErr, ok := err.(protohttp.HTTPError); ok && httpErr.Code == http.StatusForbidden {
-				websocketError(ws, websocket.ClosePolicyViolation, err)
+				msg := fmt.Sprintf("Missing authorization, visit %s to remedy", tap.TapRbacURL)
+				websocketError(ws, websocket.ClosePolicyViolation, errors.New(msg))
```

Comment:
```
err := fmt.Errorf("missing authorization, visit %s to remedy", tap.TapRbacURL)
websocketError(ws, websocket.ClosePolicyViolation, err)
```

Reply:
@siggy I like this change, but I think "Missing" should still be capitalized. We display this error with no prefix in the banner, so it's always going to be the first word and may look weird lowercase.

Reply:
Agree that capitalizing messages in the banner is preferable. I don't think the code at this layer should be concerned about that. Better to consistently return well-formed Go error messages, and let the presentation layer decide how to render them.

Reply:
Yep that is a good point. I'll remove the capitalization and let the open issue on how we want to present this error handle that!